DARPA-ASKEM/TERArium
# style: add terarium theme for keycloak login

# Description

Initial Keycloak theme for Terarium. Theme is loaded and selected upon startup (as per configuration in the Kubernetes deployment files). Refer to the README file for details on how to update/change the styling as needed.

NOTE: Initial Kubernetes startup might be slightly longer 
NOTE: Currently still using internal docker repository so only accessible within 

Resolves #26 

## Type of change


# How Has This Been Tested?

Visually verified local deployment

## To test

1.     Make sure you k8s stack is down i.e ./gateway.sh down
2.     Check out this branch
3.     Start the stack up again ./gateway.sh up
4.     Make sure that all pods are running and completed their init (kubectl get pods)
5.     Start the rest of the application and go to the login screen
6.     Should see Terarium themed login now


# Checklist:
